h2gis-functions/src/main/java/org/h2gis/functions/io/dbf/DBFEngine.java

```diff
@@ -61,7 +61,6 @@ public static void feedTableDataFromHeader(DbaseFileHeader header, CreateTableDa
             String fieldsName = header.getFieldName(i);
             final int type = dbfTypeToH2Type(header,i);
             Column column = new Column(fieldsName.toUpperCase(), type);
-            column.setPrecision(header.getFieldLength(i)); // set string length
```

Comment:
The precision of the column cannot be set ?

Reply:
It has been removed from the Column class in h2database ([here](https://github.com/h2database/h2database/blob/7707e5d42f4967e31c5c7487252adc3cd5564532/h2/src/main/org/h2/table/Column.java#L67))

Reply:
Maybe something like that
TypeInfo.getTypeInfo(type, precision, scale, null)

Reply:
Column column = new Column(fieldsName.toUpperCase(), TypeInfo.getTypeInfo(type, precision, scale, null));